content/chat/getting-started/javascript.textile

```diff
@@ -38,20 +38,24 @@ ably apps switch
 ably auth keys switch
 ```
 
-* Install "Node.js":https://nodejs.org/en version 18 or greater.
-* Install "ts-node":https://www.npmjs.com/package/ts-node to execute TypeScript files:
+* Create a new NodeJS project
 
 ```[sh]
-npm install ts-node
+npm init -y
 ```
 
-* Create a new project in your IDE and install the Ably Chat JavaScript SDK. This will also install the Ably Pub/Sub SDK as it is a dependency:
+* Install "Node.js":https://nodejs.org/en version 18 or greater.
```

Comment:
Should we recommend 22 LTS?

On that note, node 18 is now ([as of last month](https://github.com/nodejs/release#release-schedule)) no longer maintained, so we should probably drop support: remove 18 and add 24 to tests.

Reply:
ah good spot!

I think i'll change the wording here to "any LTS version of Node.js", wdyt?

I agree we should revisit support in the sdk too 👍

Reply:
Sounds good 👍